.changeset/slick-weeks-shave.md

```diff
@@ -0,0 +1,9 @@
+---
+"fluid-framework": minor
+---
+---
+"section": "tree"
+---
+Export SharedTree beta APIs from fluid-framework/beta
+
+`fluid-framework/beta` now contains the `@beta` APIs from `@fluidframework/tree/beta`.
```

Comment:
nit: could we link to docs for said APIs?

Reply:
All such APIs have their own changeset which included fluid-framework (since they were added since the last release) as an impacted package, so they will have docs and examples right next to this on in the release notes and change log. I think that's sufficient.